packages/fiori/src/themes/ShellBar.css

```diff
@@ -66,6 +66,10 @@
 	margin-inline-start: 0;
 }
 
+::slotted([ui5-toggle-button][slot="assistant"][pressed]) {
```

Comment:
Thanks or the addition. However, the hover state seems overwrittn, too, so we need to define style for it, too
```
::slotted([ui5-toggle-button][slot="assistant"][pressed]),
::slotted([ui5-toggle-button][slot="assistant"][pressed]:hover:not([active])) {
    color: var(--sapShell_Assistant_ForegroundColor);
}
```